src/ansys/fluent/core/utils/__init__.py

```diff
@@ -17,3 +18,19 @@ def load_module(module_name, file_path):
     spec.loader.exec_module(module)
     logger.info(f"Loaded module {module_name} from {file_path}")
     return module
+
+
+def get_examples_download_dir():
+    """Return the path to the examples download directory."""
+    parent_path = Path.home() / "Downloads"
```

Comment:
I've changed this from Documents to Downloads folder. This will no longer upload files to OneDrive, let me know if that is an issue.

Reply:
Yes, this is better now. Earlier it was going to OneDrive that was an issue.